### ISSUE TITLE
Address unresolved reference to excluded README files

### ISSUE THREAD

**Issue by arthur-shaw** [lsms-worldbank/adodownr#15]
When building the site, see warning in the logs: 

```
WARNING: Unable to resolve link target: articles/README.qmd
Output created: docs\index.html

Terminating existing preview server....DONE

Preparing to preview
[1/6] articles\topic1.qmd
WARNING: Unable to resolve link target: articles/README.qmd
[2/6] articles\topic2.qmd
WARNING: Unable to resolve link target: articles/README.qmd
[3/6] news.qmd
WARNING: Unable to resolve link target: articles/README.qmd
[4/6] reference\command1.qmd
WARNING: Unable to resolve link target: articles/README.qmd
[5/6] reference\command2.qmd
WARNING: Unable to resolve link target: articles/README.qmd
[6/6] reference\index.qmd
WARNING: Unable to resolve link target: articles/README.qmd

Watching files for changes
Browse at http://localhost:6350/
Stop the preview with quarto_preview_stop()Browsing http://localhost:6350
```

**Comment by arthur-shaw** [comment]
Seems like README is part of articles menu:

```
  project:
    type: website
    output-dir: docs
  website:
    title: labeller
    favicon: images/logo.png
    page-navigation: true
    navbar:
      search: true
      logo: images/logo.png
      left:
      - text: Reference
        href: reference/index.qmd
      - text: Articles
        menu:
        - href: articles/README.qmd
        - href: articles/topic1.qmd
        - href: articles/topic2.qmd
      right:
      - icon: github
        href: https://github.com/lsms-worldbank/labeller
  format:
    html:
      theme: cosmo
      toc: true

```

**Comment by arthur-shaw** [comment]
Resolved in #22
